apps/omg_watcher/lib/web/controllers/fallback.ex

```diff
@@ -44,6 +44,14 @@ defmodule OMG.Watcher.Web.Controller.Fallback do
     econnrefused: %{
       code: "get_status:econnrefused",
       description: "Cannot connect to the Ethereum node."
+    },
+    insufficient_funds: %{
+      code: "transaction.create:insufficient_funds",
+      description: "Account balance is too low to satisfy the payment."
+    },
+    too_many_outputs: %{
+      code: "transaction.create:too_many_outputs",
+      description: "Effective number of outputs exceed allowed maximum."
```

Comment:
I think the word "Effective" is a bit confusing and unnecessary here.
```suggestion
      description: "Number of outputs exceed allowed maximum."
```

Reply:
I wanted to express that we run out of available outputs trying to satisfy `payments ++ rests`, so this happens after request validation happened, at tx construction phase...

Reply:
Ok, I'm just not sure what the word 'Effective' means here. Is it that payments + fees + change exceed max outputs? Then maybe we can explicitly say that.
What I mean is that it should be obvious to the user what the problem is if they see this error.
And what should they do to fix it - remove a payment?

Reply:
Agreed that the message would say: `Total number of payments + change + fees exceed maximum allowed transaction's outputs.`